orte/mca/plm/lsf/plm_lsf_module.c

```diff
@@ -342,9 +343,9 @@ static void launch_daemons(int fd, short args, void *cbdata)
      * orterun can do the rest of its stuff. Instead, we'll catch any
      * failures and deal with them elsewhere
      */
-    if (lsb_launch(nodelist_argv, argv, LSF_DJOB_REPLACE_ENV | LSF_DJOB_NOWAIT, env) < 0) {
+    if ( (rc = lsb_launch(nodelist_argv, argv, LSF_DJOB_REPLACE_ENV | LSF_DJOB_NOWAIT, env)) < 0) {
         ORTE_ERROR_LOG(ORTE_ERR_FAILED_TO_START);
-        opal_output(0, "lsb_launch failed: %d", rc);
+        opal_output(0, "lsb_launch failed: %d, lsberrno %d : %s", rc, lsberrno, lsb_sysmsg() );
```

Comment:
How about making it a proper `opal_show_help()`?

😄 😄 😄

Reply:
Humm. That would be good. And looking at the `help-plm-lsf.txt` file it's obvious a copy from PBS/Torque so that needs to be cleaned out.

I have on my to do list to audit/cleanup the LSF components so I'll wait to do too much more until that PR, but I can at least fix the `help-plm-lsf.txt` file in this PR.

Reply:
Fixed in newest push. Thanks!